src/test/java/hellofx/fxheroes2/agg/SpriteExtractionTest.java

```diff
@@ -0,0 +1,74 @@
+package hellofx.fxheroes2.agg;
+
+import hellofx.fheroes2.agg.Agg;
+import hellofx.fheroes2.agg.Sprite;
+import hellofx.fheroes2.agg.icn.IcnKind;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
This is a separate note: shouldn't be POM changes to reference JUnit 5? I don't remember to add reference to it. May you add it as part of this PR (you can add also any other dependence if you plan to use them, like Mokito, or whatever it feels right for you).

Reply:
They are already in the `pom.xml`: https://github.com/ciplogic/TilePainter/blob/master/pom.xml#L13-L24